samcli/lib/utils/stream_writer.py

```diff
@@ -0,0 +1,37 @@
+"""
+This class acts like a wrapper around output streams to provide any flexibility with output we need
+"""
```

Comment:
I'm curious to know what is the performance hit, if we just flush all the time.

Reply:
I think it is completely negligible for the debugging, thoughts?

Reply:
yeah totally we need it for debugging. I'm talking about non-debugging cases as well. just curious.